Gems/LyShine/Code/Source/RenderGraph.cpp

```diff
@@ -51,6 +51,11 @@ namespace LyShine
     {
         m_textures[0].m_texture = texture;
         m_textures[0].m_isClampTextureMode = isClampTextureMode;
+
+#if defined(CARBONATED)
+        m_combinedVertices.reserve(1024);
```

Comment:
I'm not familiar with the detail like how many nodes we have. if we do have many then maybe reserving 1024 is too many. It makes sense converting everything in a batch once it receives all the AddPrimitive calls, if possible. Maybe batch converting in a draw operation if marked dirty in AddPrimitive. Anyway this is minor, the overhead is likley not that large.

Reply:
Yeah the overhead is really small per node.  Also, there's no easy way to batch it all once the add primitives are done, as the come in on demand.

Also, this an interim solution until [the real one is implemented](https://github.com/o3de/o3de/issues/17444) is implemented, which will be the better solution.